tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+
+2. Manually trigger the workflow
+
+3. The results produces an issue on the repository with a list of file and section existence checks, indicating whether each requirement was met or not.
+
+
+ **Review Content**
+
+ The project should include the following files and sections [(link to template)](https://github.com/DSACMS/repo-scaffolder/tree/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D):
+
+ - [ ] **README.md**
+    _An essential guide that gives viewers a detailed description of your project_
+
+    ---
+
+
+| **Section**         | **Description**                                                                                                                                                                                                                                                                                | **Included** |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| Project Description | 1-3 sentences short description of the project that can be used as a 'one-liner' to describe the repo. A best practice is using this same language as the official 'description' on GitHUb repo landing page.                                                                                  |              |
+| About the Project   | Longer-form description of the project. It can include history, background, details,problem statements,links to design documents or other supporting materials,or any other information/context that a user or contributor might be interested in.                                             |              |
+| Core Team           | This information helps with succession planning and provenance for security compliances and remediation. It helps future users and contributors understand where the code originated.                                                                                                          |              |
+| Policies            | This section is to explicitly link to Federal policies and guidelines that are required or rocommended for Federal projects to comply with, such as Accessibility (508) Interoperability, Anti-deficiency, Security, Licensing, and other policies that can vary between agencies and domains. |              |
+| Public Domain       | A best practice is to list the LICENSE under which a project is released at the bottom of the README. In most cases for Federal repos, we default to Creative Commons Zero 1.0 International(world-wide public domain.)                                                                        |              |
+
+
+- [ ] **License**
```

Comment:
This should be all capitalized

Reply:
What should be capitalized??? like the bold text: **LICENSE** <- like that?

Reply:
Yes!